.gitignore

```diff
@@ -6,6 +6,7 @@ __pycache__/
 # Django related
 media/
 static/
+staticfiles/
```

Comment:
Why staticfiles was added to gitignore? Also add dockerignore looks like dockerignore is missing